src/main/java/uk/gov/hmcts/divorce/common/event/page/SolicitorDetailsWithStatementOfTruth.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.divorce.common.event.page;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.sdk.type.Organisation;
+import uk.gov.hmcts.ccd.sdk.type.OrganisationPolicy;
+import uk.gov.hmcts.divorce.common.ccd.CcdPageConfiguration;
+import uk.gov.hmcts.divorce.common.ccd.PageBuilder;
+import uk.gov.hmcts.divorce.divorcecase.model.AcknowledgementOfService;
+import uk.gov.hmcts.divorce.divorcecase.model.Applicant;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.divorcecase.model.Solicitor;
+
+import static uk.gov.hmcts.divorce.divorcecase.model.UserRole.APPLICANT_1_SOLICITOR;
+
+@Component
+public class SolicitorDetailsWithStatementOfTruth implements CcdPageConfiguration {
+
+    @Override
+    public void addTo(final PageBuilder pageBuilder) {
+
+        pageBuilder
+            .page("SubmitAos")
+            .pageLabel("Statement of Truth - Solicitor")
+            .complex(CaseData::getAcknowledgementOfService)
+                .label("labelApplicant2SolStatementOfTruth-SOT", "## Statement of truth")
+                .mandatory(AcknowledgementOfService::getStatementOfTruth)
+                .done()
+            .complex(CaseData::getApplicant2)
+                .complex(Applicant::getSolicitor)
+                    .complex(Solicitor::getOrganisationPolicy, null, "Your firm's address or DX number")
```

Comment:
Isn't this normally a generic text field? Why are we adding the Org Policy again?